Marlin/ultralcd.cpp

```diff
@@ -1110,13 +1110,24 @@ static void lcd_control_volumetric_menu() {
 #ifdef HAS_LCD_CONTRAST
   static void lcd_set_contrast() {
     if (encoderPosition != 0) {
-      lcd_contrast -= encoderPosition;
-      lcd_contrast &= 0x3F;
+      #ifdef U8GLIB_LM6059_AF
+        lcd_contrast += encoderPosition;
```

Comment:
Well, it's one of the requests out there, at least to have an option to reverse just the menu direction. So, I just need to see if all knobs move the menu highlight in the same way, so that option will tend to behave the same on all printers.